packages/terra-worklist-data-grid/src/utils/WorklistDataGridUtils.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * The Object representation of the row selection column.
+ */
+const ROW_SELECTION_COLUMN = {
+  id: 'WorklistDataGrid-rowSelectionColumn',
+  width: 40,
+};
+
+const FIRST_NON_HEADER_ROW = 1;
+
+const copyCellContent = (event) => {
+  // Use the rendered elements to get the cell content to copy to the clipboard.
+  const currentCell = event.target.closest('td,th') || document.activeElement.closest('td,th');
+  const cellContent = currentCell.textContent;
+  if ('clipboard' in navigator) {
+    navigator.clipboard.writeText(cellContent);
+  } else {
+    document.execCommand('copy', true, cellContent);
+  }
+};
```

Comment:
This seems to be out of scope for this JIRA.  I don't believe that this implementation is correct.  It does not account for the images that can be present, for example.  I don't think there is a real benefit to trying to add iterative logic for copy while we restructure the design.